packages/@okta/vuepress-site/docs/reference/postman-collections/index.md

```diff
@@ -34,6 +34,7 @@ Import any Okta API collection for Postman from the following list:
 | Linked Objects                    | [![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/3ab2cf3f197337119d34) |
 | Log Streaming                     | [![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/cbb00ae768a0c02ea433)|
 | Mappings                          | [![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/42f528e8de2085ef2c97) |
+| MyAccount <ApiLifecycle access="ie" /><ApiLifecycle access="ea" /> | [![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/9cb68745dbf85ae3a871) |
```

Comment:
What does "ie" mean here?

Reply:
I think that is for identity engine, ea release.

Reply:
it's for Identity Engine so that it adds a label looking like this 
<img width="142" alt="image" src="https://user-images.githubusercontent.com/64581315/167161297-b755144c-2465-44b8-8523-acfe0843f1f0.png">

Reply:
Thank you!